examples/incompressible-flow/3d-periodic-hills/periodic-hills.prm

```diff
@@ -58,12 +58,12 @@ end
 #---------------------------------------------------
 
 subsection flow control
-  set enable               = true
-  set boundary id          = 0
-  set volumetric flow rate = -9.1575 # bulk velocity = -1
-  set flow direction       = 0
-  set initial beta         = 7.66
-  set verbosity            = verbose
+  set enable                  = true
+  set boundary id             = 0
+  set average velocity target = -1
```

Comment:
targetted average velocity might read better? I am not sure. The wording you chose is also nice.

Reply:
it can be only `average velocity`, shorter seems better